sim/global-types.ts

```diff
@@ -21,6 +21,41 @@ type BoostName = StatNameExceptHP | 'accuracy' | 'evasion';
 type BoostsTable = {[boost in BoostName]: number };
 type SparseBoostsTable = Partial<BoostsTable>;
 type Nonstandard = 'Past' | 'Future' | 'Unobtainable' | 'CAP' | 'LGPE' | 'Custom';
+/**
+ * Describes the acceptable target(s) of a move.
+ */
+type MoveTarget =
+	// Only relevant to Doubles or Triples, the move targets your ally only.
```

Comment:
This documentation should be in the `/** */` block, so it shows up on hover.

Reply:
I'll take your word for it. (I develop for PS on Linux so I don't use VS Code.)